jobs/integr8ly/poc-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,58 @@
+---
+
+- job:
+    name: poc-testing-nightly-trigger
+    project-type: pipeline
+    description: "Nightly pipeline trigger for QE POC cluster testing."
+    sandbox: false
+    concurrent: false
+    triggers:
+      - timed: "H H(0-3) * * *"
+    properties:
+      - build-discarder:
+          num-to-keep: 56
+      - inject:
+          properties-content: |
+            GH_CLIENT_ID=yourclientid
+            GH_CLIENT_SECRET=yourclientsecret
+            SELF_SIGNED_CERTS=false
```

Comment:
Is the self signed certs suppose to be false (I don't know, hence the question)?

Reply:
yes it is. QE POC cluster does have a valid cert.